tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,786 @@
+psds_std = psds_plot.std(axis=(0, 1))[rng]
+axes[0].plot(freqs[rng], psds_mean, color='b')
+axes[0].fill_between(
+    freqs[rng], psds_mean - psds_std, psds_mean + psds_std,
+    color='b', alpha=.2)
+axes[0].set(title="PSD spectrum", ylabel='Power Spectral Density [dB]')
+
+# SNR spectrum
+snr_mean = snrs.mean(axis=(0, 1))[rng]
+snr_std = snrs.std(axis=(0, 1))[rng]
+
+axes[1].plot(freqs[rng], snr_mean, color='r')
+axes[1].fill_between(
+    freqs[rng], snr_mean - snr_std, snr_mean + snr_std,
+    color='r', alpha=.2)
+axes[1].set(
+    title="SNR spectrum", xlabel='Frequency [Hz]',
+    ylabel='SNR', ylim=[-1, 15], xlim=[fmin, fmax])
+fig.show()
+
+###############################################################################
+# You can see that the peaks at the stimulation frequencies (12 hz, 15 hz)
+# and their harmonics are visible in both plots.
+# Yet, the SNR spectrum shows them more prominently as peaks from a
+# noisy but more or less constant baseline of SNR = 1.
+# You can further see that the SNR processing removes any broad-band power
+# differences (such as the increased power in alpha band around 10 hz),
+# and also removes the 1/f decay in the PSD.
+#
+# Note, that while the SNR plot implies the possibility of values below 0
+# (mean minus sd) such values do not make sense.
+# Each SNR value is a ratio of positive PSD values, and the lowest possible PSD
+# value is 0 (negative Y-axis values in the upper panel only result from
+# plotting PSD on a log scale).
+# Hence SNR values must be positive and can minimally go towards 0.
+# You can nicely see this at 50hz with the artifact induced by notch
+# filtering the line noise:
+# The PSD spectrum shows a prominent negative peak and average SNR approaches
+# 0.
+#
+
+
+###############################################################################
+# Subsetting data
+# ---------------
+#
+# Our processing yielded a large array of many SNR values for each trial x
+# channel x frequency-bin of the PSD array.
+#
+# For statistical analysis we obviously need to define specific subsets of this
+# array. First of all, we are only interested in SNR at the stimulation
+# frequency, but we also want to restrict the analysis to a spatial ROI. The
+# most interesting questions, however, will probably rely on comparing SNR in
+# different trials.
+#
+# Since we here have a large SNR array with all conditions, we will have to
+# find the indices of trials, channels, etc.
+# Alternatively, one could subselect the trials already at the epoching step,
+# using MNE's event information, and process different epoch structures
+# individually.
+#
+# Let's have a look at the trials with 12 hz stimulation, for now.
+#
+
+# define stimulation frequency
+stim_freq = 12.
+
+###############################################################################
+# get index for the stimulation frequency (12hz)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
too many ^'s, should match title. Title should also have capital G

Reply:
Fixed.